hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -2807,6 +2812,11 @@ func (r *NodePoolReconciler) listMachineTemplates(nodePool *hyperv1.NodePool) ([
 		if err != nil {
 			return nil, err
 		}
+	case hyperv1.OpenStackPlatform:
```

Comment:
the only cherry-pick conflict was here because now the CAPI stuff was moved into capi.go. But I ported the change here.

Reply:
no, what I meant here is that the change here was "manual" and not from the cherry-pick, because in `main` this was done in `capi.go` and in this branch it's in `nodepool_controller.go`. Otherwise, same function, same goal.